core/src/main/scala/com/salesforce/op/stages/impl/feature/Transmogrifier.scala

```diff
@@ -491,6 +492,21 @@ trait TrackInvalidParam extends Params {
   def setTrackInvalid(v: Boolean): this.type = set(trackInvalid, v)
 }
 
+/**
+ * Param that decides whether or not lengths of text are tracked during vectorization
+ */
+trait TrackTextLenParam extends Params {
```

Comment:
once this param is added - will existing models fail to load or not?

Reply:
Yes existing models will fail to load. That's why I had to re-generate the old model that we test loading with

Reply:
lets indicate this in the pr description so we wont forget to include it in our release notes